src/main/webapp/js/add-location-loader.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+function addLocation(){
+  const [messages, setMessages] = React.useState([]);
+  React.useEffect(fetchLocation, []);
+
+  // Fetch messages and add them to the page.
+  function fetchLocation() {
+    const url = "/api/add-location-loader";
```

Comment:
This URL is wrong, but it will be correct once you fix my PR comment above.

Reply:
just to confirm, no changes to this URL need to be made when I make the above pr comment's change?